esmvaltool/cmorizers/data/downloaders/datasets/esacci_soilmoisture.py

```diff
@@ -0,0 +1,47 @@
+"""Script to download ESACCI-SOIL-MOISTURE."""
```

Comment:
Could you replace this by ESACCI-SOILMOISTURE throughout the code? Otherwise the downloader is not found